core/common/src/main/java/alluxio/worker/block/BlockWorker.java

```diff
@@ -210,6 +212,13 @@ void requestSpace(long sessionId, long blockId, long additionalBytes)
    */
   void cache(CacheRequest request) throws AlluxioException, IOException;
 
+  /**
+   * load blocks into alluxio.
+   * @param request load request
+   * @return load status
+   */
+  List<List<BlockStatus>> load(LoadRequest request);
```

Comment:
This should be change to return List<BlockStatus> as well? or just LoadResponse?